OpenDreamRuntime/Objects/Types/DreamObjectSavefile.cs

```diff
@@ -122,4 +121,9 @@ public sealed class SavefileDirectory : Dictionary<string, DreamValue> { }
             Directories.Add(_currentDirPath, new SavefileDirectory());
         }
     }
+
+    // << statement
+    public override void OperatorOutput(DreamValue value) {
+        CurrentDir[_currentDirPath] = value;
```

Comment:
In BYOND this appears to do more than just set the value. It looks like it turns it into a list of some sort if there's already a value there.

For example
```
var/savefile/S = new()
S << "a"
S << "b"
world.log << S.ExportText()
```
Gives `. = "a","b"`

Reply:
Redirecting this to  OperatorIndexAssign or some fancy intermediate that sets the index to (A) value so that it's a list